docs/source/tutorials/mantid_basic_course/fitting_data/05_fitting_exercises.rst

```diff
@@ -72,30 +72,16 @@ Exercise 2
 
 where the orange line is the guess
 
-#. Remove plot guess
-#. Fit the data with the model, where the output should be something
-   similar to:
+9. Remove plot guess
```

Comment:
this is going from rst enumerated list to manually numbered, if you are using manual numbering change the enumeration from line 50 to manual numbering

Reply:
Good point